### PR TITLE
Added support for php 8.1

### PR DESCRIPTION
**Pull request by sebastiansulinski**
Updated composer to add support for php 8.1

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "homepage": "https://laminas.dev",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3|^8.0",
```
Comment (Ocramius) [inline composer.json]: Should be `^7.4 || ~8.0.0 || ~8.1.0`

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (sebastiansulinski) [reply composer.json]: @Ocramius aren't you planning to keep it maintained after 8.1? Seems a bit redundant to keep adding minor versions to compatibility entry?

Comment (Ocramius) [reply composer.json]: We just don't know how PHP 8.2 will break things: remember that PHP does not follow SemVer, and PHP 8.1 has already been quite brutal on a huge amount of our packages.

Comment (sebastiansulinski) [reply composer.json]: @Ocramius no problem - that's now been updated.

**Review by Ocramius — APPROVED** [review]
Really sorry, @sebastiansulinski - this went under my radar, but is now good to go! :+1:

**Pull request merged**